static/js/popovers.js

```diff
@@ -133,6 +133,68 @@ function show_user_info_popover(element, user, message) {
     }
 }
 
+// element is the target element to pop off of
+// user is the user whose profile to show
+// message is the message containing it, which should be selected
+function show_user_group_info_popover(element, group, message) {
+    var last_popover_elem = current_message_info_popover_elem;
+    popovers.hide_all();
+    if (last_popover_elem !== undefined
+        && last_popover_elem.get()[0] === element) {
+        // We want it to be the case that a user can dismiss a popover
+        // by clicking on the same element that caused the popover.
+        return;
+    }
+    current_msg_list.select_id(message.id);
+    var elt = $(element);
+    if (elt.data('popover') === undefined) {
+        if (group === undefined) {
+            // This is never supposed to happen, not even for deactivated
+            // users, so we'll need to debug this error if it occurs.
+            blueslip.error('Bad sender in message' + message.sender_id);
+            return;
+        }
+        var args = {
+            group_name: group.name,
+            group_description: group.description,
+            members: group.members.map(function (m) { return people.get_person_from_user_id(m);})
+            .map(function (p) {
+                return Object.assign({}, p, {
+                    presence_status: presence.get_status(p.user_id),
+                    is_active: people.is_active_user_for_popover(p.user_id),
+                    user_last_seen_time_status: user_last_seen_time_status(p.user_id),
+                });
+            })
+            .sort(function (a, b) { return a.full_name.localeCompare(b.full_name); }),
```

Comment:
This fairly complex fetching and sorting logic might be best extracted as a separate function so that we can eventually unit-test aspects of it (like the sort order).